indico/modules/events/editing/client/js/components/Timeline/judgment/TagInput.jsx

```diff
@@ -0,0 +1,77 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2019 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import {Dropdown} from 'semantic-ui-react';
+import {FinalField} from 'indico/react/forms/fields';
+import {Translate} from 'indico/react/i18n';
+
+import './TagInput.module.scss';
+
+/**
+ * A dropdown which allows tags to be searched and added
+ */
+function TagInput({onChange, value: defaultValue, placeholder, options}) {
+  return (
+    <Dropdown
+      placeholder={placeholder}
+      styleName="tag-input"
+      upward={false}
```

Comment:
isn't that the default anyway?

Reply:
If I understood correctly, if it's not set the component tries to figure out whether to open upward or downward. Upward looks pretty ugly IMHO.

Reply:
ah good point, i didn't know it was a tristate option

Reply:
ah, good point, didn't know it was a tristate option